entries/v/vaultcord.com.json

```
@@ -0,0 +1,15 @@
+{
+  "VaultCord": {
+    "domain": "vaultcord.com",
+    "tfa": [
+      "email",
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://docs.vaultcord.com/misc/multi-factor-authentication",
+    "categories": [
+      "backup",
+      "developer"
```

I understand your reasoning for adding the developer category, but many services provide users with an API and most of them are not developer tools. I think Backup & Sync is the proper category for your service.
```suggestion
      "backup"
```